springfield/firefox/firefox_details.py

```diff
@@ -86,10 +86,6 @@ def platforms(self, channel="release", classified=False):
         else:
             platforms = self.platform_labels.copy()
 
-        # Linux ARM64/AArch64 installers not available for ESR builds.
-        if channel == "esr":
-            del platforms["linux64-aarch64"]
-
```

Comment:
I'm wondering, if, with this removed here, we don't need to look into some deeper pages at this point that would emit 404 bouncer links now. (Say, the 115ESR-old fallback CTA etc.)

Reply:
Yeah the http://localhost:8000/en-US/download/all/desktop-esr/linux64-aarch64/sco/ shows a esr115-[…]aarch64 bouncer link that won't exist:/

That page deals with that in other cases already (for locales not present for given esr-old) so excluding that for arm64 is one option.

Given it's a fallback for legacy Mac&Win, and is not officially provided for Linux… an option could be hiding the esr115 fallback for linux steps completely.